src/router.ts

```diff
@@ -0,0 +1,8 @@
+import  traineeRouter  from './ controllers/trainee/index';
+import { Router } from 'express';
+
+const mainRouter = Router();
```

Comment:
define the interface of express route type

Reply:
Done